githubaction/0.1.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,27 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: githubaction-webhook
+displayName: GitHub Action Webhook Integration
+createdAt: 2021-11-17T00:00:00Z
+description: This integration shows how to invoke GitHub Action workflows leveraging Keptn's webhook service.
+logoURL: https://upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Slack_Technologies_Logo.svg/2560px-Slack_Technologies_Logo.svg.png
```

Comment:
Please change logo to a GitHub Actions logo

Reply:
Yes but  should I put an image in this repo? or does it need to be an external  URL one as was done in the slack example?

Reply:
I propose to add an image to `/images` and the reference it using: `https://raw.githubusercontent.com/keptn-sandbox/artifacthub/main/slack/0.1.0/images/xyz.png`